packages/odata-service-writer/src/types.ts

```diff
@@ -71,7 +71,7 @@ export interface OdataService {
     /**
      * Annotations can either be EDMX annotations or CDS annotations.
      */
-    annotations?: EdmxAnnotationsInfo | CdsAnnotationsInfo;
+    annotations?: EdmxAnnotationsInfo[] | CdsAnnotationsInfo[];
```

Comment:
This is a breaking type change. `@sap/fiori-generator` will need to be updated to accomodate, or better if you continue to support backward compatible single entries

Reply:
Ok, I changed it back and kept array logic.